mod/ckeditor/views/default/js/elgg/ckeditor.js

```diff
@@ -33,6 +33,9 @@ define(function(require) {
 		 * @return void
 		 */
 		init: function() {
+			// show the toggle-editor link
```

Comment:
Needs documentation on why this is done like this. Add similar explanation that you have as the PR description.